presets/generateConfig.json

```diff
@@ -0,0 +1 @@
+{"bridgeAddr":"0xdaf81c57a228909bd951e0af495a4074dd1e15d8","operatorAddr":"0x6eead7043e19dbe82594e3aa507148bbe4d52be4","exitHandlerAddr":"0x544c2fa32ec0c9b4be3526db4220499d58e61783","rootNetwork":"http://localhost:7545","network":"ganache","networkId":445129242,"peers":[]}
```

Comment:
1. What is this file for?
2. Why it's not formatted?
3. I don't think we should configure 3 addresses. We can fetch everything from one contract, isn't? It would be really easy to make mistake with 3 addresses

Reply:
The data/functionality is now split between 3 contracts. The config is generated in the contracts deployment script, hence the formatting.

Reply:
You need to configure only exitHandler contract, then you can get all addresses from it, no?

`exitHandler.bridge` and `bridge.operator`

Reply:
I could do that, but I think it is better if we keep this in the config. This would add 2 additional async calls in the setup, which adds complexity.

Reply:
Maybe it'll make code of the node a little bit more complex, but it'll reduce complexity basically in all other places (including bridge-ui).

Me trying to see complexity in 2 contract calls:
![image](https://user-images.githubusercontent.com/1477583/49342544-ac89b280-f697-11e8-9895-d4b83178023f.png)

Reply:
Btw, if this file is generated, why it's included in commit?

Reply:
Well now we construct the contracts from the config, but here I would have to construct a temporary contract, make calls, construct config and create the same contract again from the config. It just seems like a big change.

Reply:
I deleted in in latest commit.

Reply:
Well I imagine all the configs will be generated in the future, hence it should not be a big problem. I can change it if you want, but I suspect it will take a while and lead to a lot more changes throughout the code again.

Reply:
I will attempt now. I might be over-reacting, I will try and see what happens.

Reply:
few comments:
> If something can be derived, it should be derived

agree. yet there is this other wisdom: "do not use the blockchain as database". The node could be down or not synced correctly, leading to some kind of problem. there can go little wrong with reading a config file. we could still check later with the chain if the data is correct, and then halt the node, or something.

> all the configs will be generated in the future

could we extract the config generation into a separate issue (technical debt). the config would then be generated (with addresses for all contracts) on network creation and propagated through the node. bridge-ui could take an node-IP instead of an address in url to fetch all addresses from node config.

Reply:
https://github.com/leapdao/leap-node/commit/f0dd07a9bcf0f3010d51d62cd4cba8f6e3eda6e2

Reply:
I still think `BridgeState.prototype.init` is a better place for that